client/my-sites/ads/form-earnings.jsx

```diff
@@ -81,198 +71,229 @@ module.exports = React.createClass( {
 		}, 0 );
-	},
+	}
 
-	toggleEarningsNotice: function( event ) {
+	updateSettings = () => {
+		this.setState( this.getSettingsFromStore() );
+	};
+
+	handleEarningsNoticeToggle = ( event ) => {
 		event.preventDefault();
-		this.setState( { showEarningsNotice: ! this.state.showEarningsNotice } );
-	},
+		this.setState( {
+			showEarningsNotice: ! this.state.showEarningsNotice
+		} );
+	};
 
-	toggleInfo: function( type, event ) {
+	handleInfoToggle = ( type ) => ( event ) => {
 		event.preventDefault();
 		switch ( type ) {
 			case 'wordads':
-				this.setState( { showWordadsInfo: ! this.state.showWordadsInfo } );
+				this.setState( {
+					showWordadsInfo: ! this.state.showWordadsInfo,
+				} );
 				break;
 			case 'sponsored':
-				this.setState( { showSponsoredInfo: ! this.state.showSponsoredInfo } );
+				this.setState( {
+					showSponsoredInfo: ! this.state.showSponsoredInfo,
+				} );
 				break;
 			case 'adjustment':
-				this.setState( { showAdjustmentInfo: ! this.state.showAdjustmentInfo } );
+				this.setState( {
+					showAdjustmentInfo: ! this.state.showAdjustmentInfo,
+				} );
 				break;
 		}
-	},
+	};
 
-	getInfoToggle: function( type ) {
-		var types = {
+	getInfoToggle( type ) {
+		const types = {
 			wordads: this.state.showWordadsInfo,
 			sponsored: this.state.showSponsoredInfo,
 			adjustment: this.state.showAdjustmentInfo,
-		}
+		};
 
 		return types[ type ] ? types[ type ] : false;
-	},
+	}
 
-	checkSize: function( obj ) {
+	checkSize( obj ) {
 		if ( ! obj ) {
 			return 0;
 		}
 
 		return Object.keys( obj ).length;
-	},
+	}
 
-	swapYearMonth: function( date ) {
-		var splits = date.split( '-' );
+	swapYearMonth( date ) {
+		const splits = date.split( '-' );
 		return splits[ 1 ] + '-' + splits[ 0 ];
-	},
-
-	getStatus: function( status ) {
-		var statuses = {
-			0: this.translate( 'Unpaid' ),
-			1: this.translate( 'Paid' ),
-			2: this.translate( 'a8c-only' ),
-			3: this.translate( 'Pending (Missing Tax Info)' ),
-			4: this.translate( 'Pending (Invalid PayPal)' )
-		}
+	}
+
+	getStatus( status ) {
+		const { translate } = this.props;
+		const statuses = {
+			0: translate( 'Unpaid' ),
+			1: translate( 'Paid' ),
+			2: translate( 'a8c-only' ),
+			3: translate( 'Pending (Missing Tax Info)' ),
+			4: translate( 'Pending (Invalid PayPal)' ),
+		};
 
 		return statuses[ status ] ? statuses[ status ] : '?';
-	},
+	}
 
-	payoutNotice: function() {
-		var owed = this.state.earnings && this.state.earnings.total_amount_owed ? this.state.earnings.total_amount_owed : '0.00',
-			notice = this.translate(
-				'Outstanding amount of $%(amountOwed)s does not exceed the minimum $100 needed to make the payment. Payment will be made as soon as the total outstanding amount has reached $100.',
+	payoutNotice() {
+		const { translate } = this.props;
+		const owed = this.state.earnings && this.state.earnings.total_amount_owed ? this.state.earnings.total_amount_owed : '0.00',
+			notice = translate(
```

Comment:
I'm also inclined to do that, but since we don't have a clear standard for which one is better, I'll leave them as they are.